backend/src/email/routes/email-campaign.routes.ts

```diff
@@ -608,6 +611,9 @@ router.post(
   '/send',
   celebrate(sendCampaignValidator),
   CampaignMiddleware.canEditCampaign,
+  EmailMiddleware.getCampaignFromAddress,
```

Comment:
if we're checking it here right before send, do we still need to check the address's validity at each stage?

Reply:
Technically we don't need to. But I added it so that it checks at the same places as before. Main flows I tested were same as #728